src/main/java/hudson/scm/IntegrityCheckpointAction.java

```diff
@@ -92,6 +98,19 @@ public static String evalGroovyExpression(Map<String, String> env, String expres
       return result.toString().trim();
     }
   }
+  
+  /**
+	 * @author asen
+	 * Default Whitelist
+	 *
+	 */
+	@Extension
+	public static class DefaultWhitelist extends ProxyWhitelist {
+		public DefaultWhitelist() throws IOException {
+			super(new StaticWhitelist("method java.text.Format format java.lang.Object",
+					"new java.text.SimpleDateFormat java.lang.String"));

```

Comment:
Add as a PR to `script-security-plugin` if not already there.

Reply:
Dear @jglick ,
PR already submitted and closed for this. https://github.com/jenkinsci/script-security-plugin/pull/119/files. I will remove this in a subsequent plugin iteration.